dhis-2/dhis-support/dhis-support-hibernate/pom.xml

```diff
@@ -48,18 +49,18 @@
       <groupId>cglib</groupId>
       <artifactId>cglib</artifactId>
     </dependency>
-    
+
     <!-- Jasypt -->
 
     <dependency>
-	  <groupId>org.jasypt</groupId>
-	  <artifactId>jasypt</artifactId>
+      <groupId>org.jasypt</groupId>
+      <artifactId>jasypt</artifactId>
     </dependency>
     <dependency>
-	  <groupId>org.jasypt</groupId>
-	  <artifactId>jasypt-hibernate4</artifactId>
+      <groupId>org.jasypt</groupId>
```

Comment:
This dependency was removed from here, and the main project pom.xml in commit e32ba66f53911cf4e38bb124e3782d8707f584fb and is no longer in use (in master). Will fix.